tests/unit/test_oci.py

```diff
@@ -228,3 +228,109 @@ def test_set_env(self, mocker):
                 universal_newlines=True,
             )
         ]
+
+    @patch("subprocess.run")
+    @patch("tempfile.mkdtemp")
+    @patch("pathlib.Path.mkdir")
+    @patch("shutil.rmtree")
+    @patch("rockcraft.oci._compress_layer")
```

Comment:
can you use the mocker fixture instead of these?

something like https://github.com/canonical/rockcraft/pull/31/files#diff-43d03eb32b3769225898764d036e6e7fad2649ce712b0d45135d084631789acfL29 ?

Reply:
the process_run fixture is probably satisfying the need to patch subprocess.run too

Reply:
done in https://github.com/canonical/rockcraft/pull/31/commits/5f58b87e3733ec797b8db80c2adeb3c5f2bdb92a